drivers/hmis/lib/form_data/default/fragments/current_living_situation.json

```diff
@@ -64,7 +64,7 @@
       ]
     },
     {
-      "type": "CHOICE",
+      "type": "STRING",
```

Comment:
I don't think it makes sense to have a hidden "CHOICE" without any list options. This is probably just a string

Reply:
This needs implementing still, added https://github.com/open-path/Green-River/issues/6272

Reply:
Seems like it's okay to use a string until we implement a pick list for 4.12.3?

Reply:
yep!